File: Week3 - Algorithm/lecture/phonebook.c
#include <cs50.h>
#include <stdio.h>
#include <string.h>

//create object person
typedef struct{
        string name;
        string number;
}
person;

int main(void)
{

    //Phone book by two string arrays
    // string names[] = {"Carter", "David", "John"};
    // string numbers[] = {"+416-23231-213123", "12312312-232233", "21312313"};

    // string name = get_string("String: ");

    // for (int i = 0; i < 3; i++)
    // {
    //     if (strcmp(names[i],name) == 0)
    //     {
    //         printf("Found %s\n", numbers[i]);
    //         return 0;
    //     }
    // }
    // printf("NOt Found\n");
    // return 1;


    person people[3];
    people[0].name = "Carter";
    people[0].number = "123123";
    people[1].name = "David";
    people[1].number = "1232183838";
    people[2].name = "John";
    people[2].number = "4412415125";

    string name = get_string("String: ");

    for (int i = 0; i < 3; i++)
    {
        if (strcmp(people[i].name,name) == 0)
        {
            printf("Found %s\n", people[i].number);
            return 0;
        }
    }
    printf("NOt Found\n");
    return 1;
}

File: Week6 - Python/lecture/phonebook.py
# names = ["Carter", "David", "John"]

# name = input("Name: ")

# # for loop can have else
# # for n in names:
# #     if name == n:
# #         print("Found")
# #         break
# # else:
# #     print("Not Found")


# if name in names:
#     print("Found")
# else:
#     print("Not Found")


# Loop though a dictonary array
# people = [
#     {"name": "Carter", "number": "+1 23210333-3213"},
#     {"name": "David", "number": "+1 213-3213"},
#     {"name": "John", "number": "+1 213-3213"},
# ]

# name = input("Name: ")

# for person in people:
#     if person["name"] == name:
#         print(f"Found {person["number"]}")
#         break
# else:
#     print("Not Found")

people = {
    "Jacky":"123123",
    "Koey":"41244",
    "James":"53336"
}
name = input("Name: ")

if name in people:
    print(f"Found {people[name]}")
else:
    print("Not Found")
